bin/lolcommits

```diff
@@ -408,10 +332,10 @@ if not (Choice.choices[:enable] || Choice.choices[:disable])
   elsif Choice.choices[:last]
     do_last
   elsif Choice.choices[:browse]
-    die_if_not_git_repo!
-    Launchy.open configuration.loldir
+    Fatals.die_if_not_git_repo!
+    Launcher.open_folder(configuration.loldir)
   elsif Choice.choices[:gif]
-    do_gif
+    HistoryAnimator.new(configuration).run(Choice.choices[:gif])
```

Comment:
You are right, I struggled with the name on this one, I was trying to avoid "animation" because I didn't want to confuse between that and the (likely more commonly used) animated capture we support via Videosnap.  I'm going to go with the `TimelapseGif` you mention, it seems much more clear than what I came up with!